dags/datasets/files_to_datastore/indoor-ice-rinks.yaml

```diff
@@ -0,0 +1,125 @@
+# package name
+indoor-ice-rinks:
+  # DAG info
+  schedule: "@once" #"15 19 * * *"
+  dag_owner_name: Mackenzie
+  dag_owner_email: mackenzie.nichols4@toronto.ca
+
+  # mandatory package attributes
+  title: "Indoor Ice Rinks"
+  date_published: "2019-07-23 17:37:01.74896"
+  refresh_rate: Daily
```

Comment:
Just to confirm, please ignore if "Daily" is the correct frequency. :)

Reply:
ah! good catch! I will change that now